apps/dashboard/src/lib/redis.ts

```diff
@@ -0,0 +1,21 @@
+import "server-only";
+import Redis from "ioredis";
+
+const { REDIS_URL } = process.env;
```

Comment:
This will probably not work when you create a build - `REDIS_URL` will be `undefined`

Reply:
fixed already :)